pkg/handler/invites_test.go

```diff
@@ -0,0 +1,262 @@
+package handler
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/defipod/mochi/pkg/config"
+	"github.com/defipod/mochi/pkg/entities"
+	"github.com/defipod/mochi/pkg/logger"
+	"github.com/defipod/mochi/pkg/repo/pg"
+	"github.com/defipod/mochi/pkg/request"
+	"github.com/defipod/mochi/pkg/util/testhelper"
+	"github.com/gin-gonic/gin"
+	"github.com/stretchr/testify/require"
+)
+
+func TestHandler_GetInviteTrackerConfig(t *testing.T) {
+	db := testhelper.LoadTestDB("../../migrations/test_seed")
+	repo := pg.NewRepo(db)
+	cfg := config.LoadTestConfig()
+	log := logger.NewLogrusLogger()
+	entity := entities.New(cfg, log, repo, nil, nil, nil, nil, nil, nil, nil, nil)
+
+	h := &Handler{
+		entities: entity,
+		log:      log,
+	}
+
+	tests := []struct {
+		name             string
+		query            string
+		wantCode         int
+		wantResponsePath string
+	}{
+		{
```

Comment:
can add empty config invite ?

Reply:
You mean test with guild does not have config? It's "404_guild_id_does_not_exist" testcase